service/bitbucket/annotator.go

```diff
@@ -96,6 +90,8 @@ func (r *ReportAnnotator) Post(_ context.Context, c *reviewdog.Comment) error {
 	return nil
 }
 
+func (*ReportAnnotator) ShouldPrependGitRelDir() bool { return true }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method ReportAnnotator.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChBiYzhjMTRhZTk1ZjEzNmJhEg9nb2xpbnQtY2lyY2xlY2k= -->